### DISCUSSION TITLE
gumtree dotdiff truncating the label?

### DISCUSSION THREAD

**Discussion by mahtab-nejati** [GumTreeDiff/gumtree#301] [Q&A]
It seems that the `dotdiff` command truncates the node label to a specific length. Is there some way to change this length or generate the diff with full labels (without changing the code [here](https://github.com/GumTreeDiff/gumtree/blob/3c4d12602bf30a808c5937062592075ea79ab3d3/client.diff/src/main/java/com/github/gumtreediff/client/diff/dotdiff/DotDiff.java#L106))?

**Comment by jrfaller** [comment]
Right now it's a constant in the code ;-)

Maybe we could set a global option for that, WDYT ?

**Reply by mahtab-nejati** [reply]
That can actually be useful. I believe the labels are truncated for the sake of the readability of the output. In my case, I need to process the output and need the full label for that...

**Reply by jrfaller** [reply]
OK, so maybe you can post an issue w.r.t. this feature! However I cannot promise to work on it right away 😄

**Reply by mahtab-nejati** [reply]
I am currently modifying the code to accomplish the task, so there's no rush. I'll start an issue about this. Thanks for being so responsive.
